demo/summer-2016.html

```diff
@@ -242,7 +242,22 @@
         enableLiveAutocompletion: true
     });
 
-
+  var HashHandler = require("ace/keyboard/hash_handler").HashHandler;
+  keyboardHandler = new HashHandler();
+  keyboardHandler.addCommand({
+  name: 'item.',
+  bindKey: {win: 'Enter', mac: 'Enter'},
+  exec: function(editor) {
+        if(editor.getSession().getContext(editor.getCursorPosition().row) == "list") {
+          editor.insert("\n\t\\item ")
```

Comment:
it now adds `\item` nearly everywhere, e.g. if you hit Enter just after `\begin{document}`